@kiva/kv-components/vue/KvSelect.vue

```diff
@@ -58,21 +58,32 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		/**
```

Comment:
We didn't need this extra prop in the vue2 version. I think we can just use the value prop as before.